addons/overheating/ACE_Arsenal_Stats.hpp

```diff
@@ -0,0 +1,21 @@
+class EGVAR(arsenal,stats) {
+    class statBase;
+    class ACE_allowSwapBarrel: statBase {
```

Comment:
Would add condition property here with constant text property instead. IMO this only needs to be shown if the weapon actually supports barrel swapping

Reply:
I'd like it to show if they can't be swapped too. For example the PKP is an upgraded PKM and if my community that's resulted in people assuming you can swap the barrel when it's actually not possible to do on the PKP.